### PR TITLE
Altered bower dependencies to fix major releases breaking changes conflicts and Travis script

### PR DESCRIPTION
**Pull request by larslemos**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ before_deploy:
   - sed -i "s/__RELEASE_VERSION__/$TRAVIS_TAG/g" bintray-release.json
 deploy:
   - provider: bintray
-    file: "bintray.json"
+    file: "bintray.json"9
```
Comment (valvijo) [inline .travis.yml]: Is this "9" intentional?

File: test/karma.conf.js [hunk 2]
```diff
@@ -74,6 +74,7 @@ module.exports = function (config) {
       'lib/modernizr.custom.80690.js',
       // application files
       'app/**/*.js',
+      'app/home/**/*.js',
```
Comment (valvijo) [inline test/karma.conf.js:77]: Is this line really needed? The query above it is inclusive

Comment (larslemos) [reply test/karma.conf.js:77]: Well, it's supposed to be all included on 'app/**/*.js',
But the moduels where not leading either way 
so  'app/home/*.js', and 'app/coomon/*.js', helped.
I think there is a problem in the order/priority

File: .travis.yml (hunk 1 above)
Comment (larslemos) [reply .travis.yml]: Nop, I corrected in the latter comming

**Pull request merged**